rfcs/text/formik.md

```diff
@@ -0,0 +1,624 @@
+- Start Date: 2019-11-15
+
+
+# Table of contents
+
+- [Table of contents](#table-of-contents)
+- [Summary](#summary)
+- [Motivation](#motivation)
+- [Detailed design](#detailed-design)
+  - [Overview of Carbon's form and validation functionality](#overview-of-carbons-form-and-validation-functionality)
+  - [Overview of Formik](#overview-of-formik)
+    - [Form validation using Formik](#form-validation-using-formik)
+  - [Code examples](#code-examples)
+    - [Checkbox and CheckboxGroup](#checkbox-and-checkboxgroup)
+      - [HTML and React](#html-and-react)
+      - [Checkbox usage with Formik](#checkbox-usage-with-formik)
+      - [Form-level validation examples](#form-level-validation-examples)
+      - [Simplification using `<Field>`](#simplification-using-field)
+      - [Field-level validation example](#field-level-validation-example)
+      - [CheckboxGroup example: uncontrolled](#checkboxgroup-example-uncontrolled)
+      - [CheckboxGroup example: controlled](#checkboxgroup-example-controlled)
+    - [Blocking navigation on unsaved changes](#blocking-navigation-on-unsaved-changes)
+- [Drawbacks](#drawbacks)
+  - [No support for non-blocking ("warning" or "info") validations](#no-support-for-non-blocking-%22warning%22-or-%22info%22-validations)
```

Comment:
@edleeks87: This change you made in 6f93191 seems to have broken the link:

```diff
- [No support for non-blocking ("warning" or "info") validations](#no-support-for-non-blocking-warning-or-info-validations)
+ [No support for non-blocking ("warning" or "info") validations](#no-support-for-non-blocking-%22warning%22-or-%22info%22-validations)
```

This link doesn't work for me:

* https://github.com/Sage/carbon/blob/FE-2283-formik-rfc/rfcs/text/formik.md#no-support-for-non-blocking-%22warning%22-or-%22info%22-validations

But the original link works for me:

* https://github.com/Sage/carbon/blob/FE-2283-formik-rfc/rfcs/text/formik.md#no-support-for-non-blocking-warning-or-info-validations

Please can we revert the change? :point_down:

```suggestion
  - [No support for non-blocking ("warning" or "info") validations](#no-support-for-non-blocking-warning-or-info-validations)
```

Reply:
docs: revert link format